webapp/packages.py

```diff
@@ -53,6 +76,10 @@ def process_list(self, api_version, data): # pylint: disable=unused-argument
         if not packages:
             return packagelist
 
+        if len(packages) == 1:
+            # treat single-label like a regex, get all matching names
+            packages = self.find_packages_by_regex(packages[0])
```

Comment:
this doesn't work, on next line is `for pkg in packages:` and it produces something like this:

```
curl -X GET http://localhost:8080/api/v1/packages/pam_ssh_agent_auth-0.9-70.el6.x86_64
{"package_list": {"p": {}, "a": {}, "m": {}, "_": {}, "s": {}, "h": {}, "g": {}, "e": {}, "n": {}, "t": {}, "u": {}, "-": {}, "0": {}, ".": {}, "9": {}, "7": {}, "l": {}, "6": {}, "x": {}, "8": {}, "4": {}}}
```